backend/src/serverless/microservices/python/crowd-members-score/crowd/members_score/worker.py

```diff
@@ -6,4 +6,4 @@ def members_score_worker(tenant_id):
 
 
 if __name__ == "__main__":
-    members_score_worker("ab1d6621-9d56-411a-92ad-cd24a2185cd7")
```

Comment:
Why is this uuid hardcoded to begin with?

Reply:
Everything under `if __name__ === "__main__"` only gets executed when calling the file directly, so this will never run. But will remove anyway 😄